templates/charms-architecture.html

```diff
@@ -0,0 +1,191 @@
+    </div>
+    <div class="u-fixed-width col-start-large-4" style="padding-top: 1rem;">
+      <hr class="p-rule--muted">
+    </div>
+    <div class="col-3 col-start-large-4">
+      <p class="p-heading--5">3</p>
+    </div>
+    <div class="col-9 col-start-large-7">
+      <p>The Juju agents running on the charm constantly monitor the controller, looking for changes that affect them. When there is a discrepancy between their local state and the one represented in the controller they compute a diff and create some “events”.</p>
+    </div>
+    <div class="u-fixed-width col-start-large-4" style="padding-top: 1rem;">
+      <hr class="p-rule--muted">
+    </div>
+    <div class="col-3 col-start-large-4">
+      <p class="p-heading--5">4</p>
+    </div>
+    <div class="col-9 col-start-large-7">
+      <p>The agent then executes the charm code, which includes a specific set of actions that need to occur when a specific event occurs, as well as logic to specify their relative ordering.</p>
+    </div>
+    <div class="u-fixed-width col-start-large-4" style="padding-top: 1rem;">
+      <hr class="p-rule--muted">
+    </div>
+    <div class="col-3 col-start-large-4">
+      <p class="p-heading--5">5</p>
+    </div>
+    <div class="col-9 col-start-large-7">
+      <p>The charm operates the workload in whatever way appropriate to the event being handled; if necessary, it will interact with the live workload through pebble, to read/write the workload filesystem, run commands, etc…</p>
+    </div>
+    <div class="u-fixed-width col-start-large-4" style="padding-top: 1rem;">
+      <hr class="p-rule--muted">
+    </div>
+    <div class="col-3 col-start-large-4">
+      <p class="p-heading--5">6</p>
+    </div>
+    <div class="col-9 col-start-large-7">
+      <p>The remote state is aligned with the one represented in the controller.</p>
+    </div>
+    <div class="col-start-large-4" style="padding-top: 1rem;">
+      <p>For more detailed information you can read our <a href="/docs/sdk/charm-lifecycle">charm lifecycle documentation article&nbsp;&gt;</a></p>
+    </div>
+    <div class="col-start-large-4" style="padding-top: 2rem;">
+      <p>The Juju controller persists a wide ranging set of information about the status of the system and the deployed charms, including:</p>
+      <ul>
+        <li>Integration data</li>
+        <li>charm configuration</li>
+        <li>unit/application status</li>
+        <li>charm state</li>
+        <li>leadership status</li>
+      </ul>
+      <p>Charms at runtime can read/write some data from the controller database and thereby access some configuration parameters to decide which code path to execute and how to manage its workload.</p>
+    </div>
+  </div>
+</section>
+<section class="p-strip u-no-padding--top">
```

Comment:
Strips should only be used if there is a need for both top and bottom padding (usually when background changes). If there is only a need for bottom padding, use section instead: https://vanillaframework.io/docs/patterns/section